### DISCUSSION TITLE
Newbie here!

### DISCUSSION THREAD

**Discussion by carlosilipo77** [pimcore/pimcore#15537] [General]
I am really confused with interface. Any suggestions would be helpful. The PacketPub book - is that still current?

it is different from Wordpress/drupal interface.

the YouTube vids just ok!

thanks
Carlo 
Sorry if posted on the wrong channel.

**Comment by elburro1887** [comment]
Good luck getting any help here. The maintainers seem to not really give a damn about any of these discussions here.

**Reply by dpfaffenbauer** [reply]
why so negative here? your only post did get an answer

**Reply by elburro1887** [reply]
Where my negativity comes from?

I have been a heavy Pimcore user for 5 years, have implemented a few sites with it and have read my fair share of the GitHub issues and discussion forums in this time, and I see how the main people interact with the questions here. This user posted a question 2 weeks ago and never received an answer. I also posted a question months ago and still no answer from anybody. 

I love Pimcore and I use it on a lot of projects, but the way the maintainers interact with the community is pretty lacking, to be honest. Most questions are not answered, and a lot of them are answered with "Why don't you do a PR ;) ?", which is also a pretty useless answer, since most people just want to know if a feature is even available or if the Pimcore team will implement the feature or not, and will not want to spend time doing PRs. That's just a simple fact. 

And OK, sorry for the "ran" post above, but my point still stays. Pimcore is a great piece of software with great potential, but there still a lot that could be done in point of community engagement and support.

**Reply by fashxp** [reply]
Let's put some things into perspective here: 

Pimcore is an open source platform and you can use it free of charge. We are investing a lot of time and effort into it and just giving it to you without earning anything by doing so. 
So it might be obvious, that we also have some work to do which actually helps us earning money and paying our bills, and sadly are not able to spent day and night in the forum. 

Besides that: This forum is a community forum and the community is all of us - so everybody is invited to participate and answer questions. I think this isn't working too bad, there are a bunch of really active and helpful Pimconauts out there, which we really appreciate 😍👏.

The answer "Why don't you do a PR ;) ?" actually says a lot: no, that feature is not available yet, but we would like it and if somebody is ready to contribute, we are ready to review, work together and merge it. Pretty everything you need to know, I would say.

If you need more support and guidance, there are several professional offers you can choose from. 

Thank you for your understanding and your dedication to Pimcore. Let's work together and make it even better.

**Reply by elburro1887** [reply]
Thanks for your reply.

I still need to say one last thing here though - of course you are earning money here, the new and current open source users are the future paying enterprise customers, and people who pay for pimcore certificates, etc... You also received a series B investment last year, which validated your good work!
So you are gaining a large value from more people using your open source software. And that's fine. Everyone wants the platform to thrive and be successful since they don't want to be left with a abandoned pile of stuff in a few years.  

Anyways, my suggestion would be to improve the developer experience and the relations a bit more, in order to 
build up a more vibrant community and so that the current and new people who are interested and passionate about Pimcore are not scared away.

**Comment by plixus** [comment]
Hi @carlosilipo77 , I think this Udemy course is really helpful:
https://www.udemy.com/course/2022-learning-pimcore-from-zero-to-hero/
Preview some parts on Youtube: https://youtube.com/playlist?list=PLI7mZwMfYH8kzMIkPrbinUvOTUbb2HBlK
(The courses are on sale regularly – I think I bought it for once for 12€.)

In this YouTube playlist you can find some good explanations about backend usage for users – "unfortunately" only in german:
https://youtube.com/playlist?list=PLFSjNb92-QheNUTgLP08PGUB3-wZ-yUWO

Also, of course, it's always a good idea to read the documentation:
https://pimcore.com/docs/platform/Pimcore/
